ecosystem/sep-0006.md

```diff
@@ -67,6 +67,50 @@ Valid CORS headers are necessary to allow web clients from other sites to use th
+* For `/deposit`
+    * If the issuer's `/deposit` endpoint immediately returns success:
+        * display the deposit info that came back from the endpoint to the user, including fee and you're done! The user will execute the deposit exernally using the instructions if they want to.
+    * If the issuer responds with an interactive flow, handle it as [described in detail](#3-customer-information-needed-interactive). We expect this is the most common flow, and it should work smoothly.
+    * For now, do not handle the non-interactive or status responses to the `/deposit` request.
+    * Handle the [special cases](#special-cases), they're relatively common.
+* For `/withdraw`
+    * If the issuer's `/withdraw` endpoint immediately returns success:
+        * Provide an interface to the user that allows them to pick a withdrawal amount and view the total fee. The user can then confirm and your wallet will initiate the withdrawal by sending the Stellar payment to the address provided by the issuer.
+    * If the anchor responds with an interactive flow, handle it as [described in detail](#3-customer-information-needed-interactive).
+    * For now, do not handle the non-interactive or status responses to the `/withdraw` request.
+* It provides a better experience for users to show deposits or withdrawal they've completed in the past via the `/transactions` endpoint, but it's not strictly necessary.
+
+### Basic Anchor Implementation
+
+* Provide a full-featured implementation of `/info`. The more info you provide there, the better a wallet can relay deposits and withdrawals from the user to you.
+* Decide which endpoints, if any, need to be authenticated, and declare that properly in the `/info` endpoint.
+* Pick your approach to [fees](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#fee). We strongly recommend using `/info` to express fees if possible as it provides a better user experience (the user can see the fee structure in the wallet early in the process).
+* If you need no KYC info, explicit user signup, or other interaction from users to show deposit info or process a withdrawal, you can have your `/deposit` and `/withdraw` endpoints immediately return success with all info that the user will need to complete the deposit or withdrawal.
+* Otherwise, we recommend you provide an interactive deposit or withdrawal flow.
+* For both interactive deposit and withdrawal:
+    * Use the [Customer info needed response](#3-customer-information-needed-interactive) and include the `id` field so the wallet can check up on the status of the transaction (using the `id`) if they want.
+    * You should almost certainly use [authentication](#authentication) in the interactive flow, so be sure to accept the [`jwt` parameter](#adding-parameters-to-the-url) so the user does not have to re-login. Support the `callback` parameter as well.
+* Interactive deposit
```

Comment:
Same comment as above: I think this will be easier to read if "For both interactive deposit and withdrawal," "Interactive deposit," and "Interactive withdrawal" function as delineators.